app/controllers/application_controller.rb

```diff
@@ -110,4 +110,8 @@ def non_maintenance_bypass_ip_address?
     bypass_ip_list = bypass_ips.split(",").map { |ip| IPAddr.new ip.strip }
     bypass_ip_list.none? { |ip| ip.include?(user_ip(request.env.fetch("HTTP_X_FORWARDED_FOR", ""))) }
   end
+
+  def current_form
+    @current_form ||= Form.find(params[:form_id])
```

Comment:
Will this return anything if it's called somewhere that doesn't have a `:form_id` param? Or will it throw an error?

Reply:
It will throw an error because the find method needs a Hash and if params doesn't have :form_id it would be nil. 

>raise ArgumentError, "expected attributes to be able to convert to Hash, got #{attributes.inspect}"

Reply:
Cool, I think that's what I'd hope would happen (rather than it failing silently)